File: benchmarks/fib42/fib42.go
package main

func fib(n int64) int64 {
    if n <= 1 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    n := int64(42)
    result := fib(n)
    println(result)
}

File: benchmarks/fib42/fib42.rs
fn fib(n: u64) -> u64 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let n: u64 = 42;
    let result = fib(n);
    println!("{}", result);
}
